### PR TITLE
[BLOG] Add enhancements-to-numba-guvectorize-decorator

### PR DESCRIPTION
**Pull request by MarsBarLee**
Link to [blog post, successful build](https://labs-git-enhancements-to-numba-guvectorize-decorator-quansight.vercel.app/blog/enhancements-to-numba-guvectorize-decorator).

## Links to other Quansight blogs:
There is no need to check for Blog2Blog links due to updated website build. ([link to details in updated Labs blog migration process document](https://www.notion.so/quansightlabs/Labs-blog-migration-process-200c3297b67245b3953dd23487badfe9#af61137685c849c0affcdbab853282a7))

## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

File: apps/labs/posts/enhancements-to-numba-guvectorize-decorator.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+---
+title: 'Enhancements to Numba''s guvectorize decorator'
+published: February 25, 2021
+author: guilherme-leobas
+description: 'Starting from Numba 0.53, Numba will ship with an enhanced version of the @guvectorize decorator. Similar to the @vectorize decorator, @guvectorize now has two modes of operation: Eager, or decoration-time compilation and Lazy, or call-time compilation'
+category: [PyData Ecosystem]
```
Comment (noatamir) [inline apps/labs/posts/enhancements-to-numba-guvectorize-decorator.md:6]:
```suggestion
category: [PyData ecosystem]
```

Comment (noatamir) [reply apps/labs/posts/enhancements-to-numba-guvectorize-decorator.md:6]: Doesn't show up in the filter with the E 🤷🏽‍♀️

**Review by noatamir — APPROVED** [review]
LGTM other than 1 comment

File: apps/labs/posts/enhancements-to-numba-guvectorize-decorator.md (hunk 1 above)
Comment (MarsBarLee) [reply apps/labs/posts/enhancements-to-numba-guvectorize-decorator.md:6]: I'll mark `[PyData Ecosystem]` → `[PyData ecosystem]` with the lowercase 'e' in the [Labs blog migration document](https://www.notion.so/quansightlabs/Labs-blog-migration-process-200c3297b67245b3953dd23487badfe9) to catch it next time.

Is this category the only one that needs a case sensitive fix?

**Pull request merged**